tests/textcase/lsp/lsp_spec.lua

```diff
@@ -44,5 +44,18 @@ describe("LSP", function()
       local expected_code = test_helpers.read_file("./tests/textcase/lsp/fixtures/snake-case.ts")
       assert.are.same(table.concat(content, "\n"), expected_code)
     end)
+
+    it("Should be triggered on keybinding for constant case", function()
+      test_helpers.execute_keys("/variable_to_be_tested<CR>gaN")
+      local content = nil
+      test_helpers.wait_for(5 * 1000, function()
```

Comment:
This await check is a good candidate to become a function (in the future). [WET](https://kentcdodds.com/blog/aha-programming) is totally fine

Reply:
True!